packages/core/src/actions/types.ts

```diff
@@ -160,19 +160,16 @@ export type ProposalRequest = {
   apiKey: string
   orgId: string
   isTestnet: boolean
-  owners: string[]
-  threshold: number
   safeAddress: string
   moduleAddress: string
-  safeInitData: string
-  safeInitSaltNonce: string
-  deploymentName: string
+  projectName: string
   chainIds: Array<number>
   projectDeployments: Array<ProjectDeployment>
   gasEstimates: Array<NetworkGasEstimate>
   diff: SphinxPreview
   compilerConfigId: string | undefined
   deploymentConfigId: string | undefined
+  sphinxPluginVersion: string | undefined
```

Comment:
We send this value to the propose API endpoint so we can track it in analytics. This will help us better understand when we can safely deprecate things in the propose API endpoint.